modules/ppcp-settings/src/SettingsModule.php

```diff
@@ -638,6 +639,21 @@ static function () use ( $container ) {
 		assert( $gateway_redirect_service instanceof GatewayRedirectService );
 		$gateway_redirect_service->register();
 
+		// Do not render Pay Later messaging if "Save PayPal and Venmo" setting is enabled.
+		add_filter(
+			'woocommerce_paypal_payments_should_render_pay_later_messaging',
+			function() use ( $container ) {
+				$settings_model = $container->get( 'settings.data.settings' );
+				assert( $settings_model instanceof SettingsModel );
+
+				if ( $settings_model->get_save_paypal_and_venmo() ) {
+					return false;
+				}
+
+				return true;
```

Comment:
This can be simplified like this: `return ! $settings_model->get_save_paypal_and_venmo();`